Jest: cannot find module 'ai/rsc'

### Description

I tried to implement test using the generated app with

`
npx create-next-app@latest --example with-jest with-jest-app
`

Then I use the example here https://sdk.vercel.ai/docs/getting-started/nextjs-app-router

But when I launch

`
yarn test
`

I have this error

```
yarn test
 FAIL  app/page.test.tsx
  ● Test suite failed to run

    Cannot find module 'ai/rsc' from 'app/page.tsx'

    Require stack:
      app/page.tsx
      app/page.test.tsx

      12 |   return (
      13 |     <div className="flex flex-col w-full max-w-md py-24 mx-auto stretch">
    > 14 |       {data && <pre>{JSON.stringify(data, null, 2)}</pre>}
         |              ^
      15 |       {messages.map((m, i) => (
      16 |         <div key={i} className="whitespace-pre-wrap">
      17 |           {m.role === 'user' ? 'User: ' : 'AI: '}

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (app/page.tsx:14:14)
      at Object.<anonymous> (app/page.test.tsx:9:54)

(node:3728) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.473 s
Ran all test suites related to changed files.

Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.

✨  Done in 191.55s.
```

Thank you for your help

### Code example

_No response_

### Additional context

_No response_

Note: It works with the Vitest version

--- a/content/docs/09-troubleshooting/60-jest-cannot-find-module-ai-rsc.mdx
+++ b/content/docs/09-troubleshooting/60-jest-cannot-find-module-ai-rsc.mdx
@@ -1,6 +1,6 @@
 ---
 title: Jest: cannot find module 'ai/rsc'
-description: Troubleshooting errors related to the "Jest: cannot find module 'ai/rsc'" error.
+description: Troubleshooting AI SDK errors related to the "Jest: cannot find module 'ai/rsc'" error.
 ---
 
 # Jest: cannot find module 'ai/rsc'

--- a/content/docs/09-troubleshooting/60-jest-cannot-find-module-ai-rsc.mdx
+++ b/content/docs/09-troubleshooting/60-jest-cannot-find-module-ai-rsc.mdx
@@ -1,0 +1,20 @@
+---
+title: Jest: cannot find module 'ai/rsc'
+description: Troubleshooting errors related to the "Jest: cannot find module 'ai/rsc'" error.
+---
+
+# Jest: cannot find module 'ai/rsc'
+
+## Issue
+
+I am using AI SDK RSC and am writing tests for my RSC components with Jest.
+
+I am getting the following error: `Cannot find module 'ai/rsc'`.
+
+## Solution
+
+Configure the module resolution via `jest config update` in `moduleNameMapper`:
+
+```
+moduleNameMapper: { ... '^ai/rsc$': '<rootDir>/node_modules/ai/rsc/dist', }
+```

